packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/SeaportCreateListing.tsx

```diff
@@ -0,0 +1,219 @@
+        <Alert variant="warning">
+          Note: This method only returns a signed message, it does not submit an order to the orderbook.
+        </Alert>
+        {transactionError
+        && (
+        <Alert variant="danger" style={{ wordBreak: 'break-word' }}>
+          {transactionError}
+        </Alert>
+        )}
+        <Form onSubmit={handleSubmit} className="mb-4">
+          <Form.Group className="mb-3">
+            <Form.Label>
+              Seaport Contract Address
+            </Form.Label>
+            <Form.Control
+              required
+              disabled
+              type="text"
+              value={seaportContractAddress}
+            />
+          </Form.Group>
+          <Form.Group className="mb-3">
+            <Form.Label>
+              NFT Contract Address
+            </Form.Label>
+            <Form.Control
+              required
+              type="text"
+              value={NFTContractAddress}
+              isValid={transaction && !transactionError}
+              isInvalid={!!transactionError}
+              onChange={(e) => setNFTContractAddress(e.target.value)}
+            />
+          </Form.Group>
+          <Form.Group className="mb-3">
+            <Form.Label>
+              Token ID
+            </Form.Label>
+            <Form.Control
+              required
+              type="number"
+              value={tokenId}
+              isValid={transaction && !transactionError}
+              isInvalid={!!transactionError}
+              onChange={(e) => setTokenId(e.target.value)}
+            />
+          </Form.Group>
+          <Form.Group className="mb-3">
+            <Form.Label>
+              Currency Amount
+            </Form.Label>
+            <Form.Control
+              required
+              type="text"
+              value={buyAmount}
+              isValid={transaction && !transactionError}
+              isInvalid={!!transactionError}
+              onChange={(e) => setBuyAmount(e.target.value)}
+            />
+          </Form.Group>
+          <Form.Group className="mb-3">
+            <Form.Label>
+              Currency Type
+            </Form.Label>
+            <Form.Select
+              required
+              value={buyType}
+              onChange={(e) => setBuyType(e.target.value as TokenType)}
+            >
+              <option value="NATIVE">Native</option>
+              <option value="ERC20">ERC20</option>
+            </Form.Select>
+          </Form.Group>
+          {buyType === 'ERC20' && (
+          <Form.Group className="mb-3">
+            <Form.Label>
+              Token Contract Address
+            </Form.Label>
+            <Form.Control
+              required
+              value={tokenContractAddress}
+              isValid={transaction && !transactionError}
+              isInvalid={!!transactionError}
+              onChange={(e) => setTokenContractAddress(e.target.value)}
+            />
+          </Form.Group>
+          )}
```

Comment:
Nitpick: Can I suggest that we display the `Type` before the amount?

```suggestion
          <Form.Group className="mb-3">
            <Form.Label>
              Currency Type
            </Form.Label>
            <Form.Select
              required
              value={buyType}
              onChange={(e) => setBuyType(e.target.value as TokenType)}
            >
              <option value="NATIVE">Native</option>
              <option value="ERC20">ERC20</option>
            </Form.Select>
          </Form.Group>
          {buyType === 'ERC20' && (
          <Form.Group className="mb-3">
            <Form.Label>
              Currency Contract Address
            </Form.Label>
            <Form.Control
              required
              value={tokenContractAddress}
              isValid={transaction && !transactionError}
              isInvalid={!!transactionError}
              onChange={(e) => setTokenContractAddress(e.target.value)}
            />
          </Form.Group>
          )}
          <Form.Group className="mb-3">
            <Form.Label>
              Currency Amount
            </Form.Label>
            <Form.Control
              required
              type="text"
              value={buyAmount}
              isValid={transaction && !transactionError}
              isInvalid={!!transactionError}
              onChange={(e) => setBuyAmount(e.target.value)}
            />
          </Form.Group>
```

Reply:
![image](https://github.com/immutable/ts-immutable-sdk/assets/142861450/eff6aa4f-b09a-4380-b8c3-b9e4f0fbd4a4)
It looks a little strange, but that might just be me. If you're all good with this we can post it though

Reply:
What's the concern there? Does renaming `Token Contract Address` to `Currency Contract Address` alleviate your concerns?

Reply:
Yeah I think that might make more sense, keeps the consistency with naming. I'll change it and send it